### PR TITLE
New Files Added:Assingment5 and assingment6

### PR DESCRIPTION
**Pull request by vinod2406**

### CODE REVIEW

File: vinod/html/ASSIN6/style2.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#main {
+    background-color: #ffffff;
+    width: 280px;
+    margin: 50px auto;
+    border: 0px solid;
+}
```
Comment (Pranjali1999) [inline vinod/html/ASSIN6/style2.css:6]: After closing this bracket, you should give space.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

File: vinod/html/ASSIN6/Assingment 6.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <lable for="Skill set">Skill Set:</lable>
+        <input type="checkbox" id="HTML" name="Skill Set" value="HTML">
+        <label for="HTML">html</label>
+        <input type="checkbox" id="Java Script" name="Skill Set" value="Java Script">
+        <label for="Java Script"> Java Script</label>
+        <input type="checkbox" id="Java" name="Skill Set" value="Java">
+        <label for="Java"> Java</label>
+        <input type="checkbox" id="Oracal" name="Skill Set" value="Oracal">
+        <label for="Oracal"> Oracal</label>
+
```
Comment (shivling123) [inline vinod/html/ASSIN6/Assingment 6.html]: remove space

File: vinod/html/ASSIN6/Assingment 6.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+   
+   </style>
+   </head>
+   <body>
+   <form>
+        <div class="block1">
+          <h1 style="text-align: right;"><em>MI-9 Intern HTMLAssesment</em></h1>
+        </div>
+         <hr>
+
```
Comment (shivling123) [inline vinod/html/ASSIN6/Assingment 6.html]: remove extra space

File: vinod/html/ASSIN6/Assingment 6.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        <label for="Java"> Java</label>
+        <input type="checkbox" id="Oracal" name="Skill Set" value="Oracal">
+        <label for="Oracal"> Oracal</label>
+          
+        
+    </div>
+  </div>
+  </form>
+    
+
```
Comment (shivling123) [inline vinod/html/ASSIN6/Assingment 6.html]: remove extra space

File: vinod/html/ASSIN6/Assingment 6.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+
+
+
+
+</body>
+</html>
+
+
```
Comment (shivling123) [inline vinod/html/ASSIN6/Assingment 6.html]: remove space

**Review by vishnukopnar — APPROVED** [review]

**Review by Akash-ing — APPROVED** [review]

**Review by Pranjali1999 — APPROVED** [review]

**Review by akash19972306 — APPROVED** [review]

File: vinod/html/ASSIN5/style3.css [hunk 6]
```diff
@@ -0,0 +1,58 @@
+body{
+    background-image: url("SS.jpg");
+   background-size: cover;
+   background-position: center;
+   font-family: 'Baloo Bhaina 2', cursive;
+   margin:0%;
+   padding:0%;
+}
+
```
Comment (SnehaMakade) [inline vinod/html/ASSIN5/style3.css]: remove extra line

File: vinod/html/ASSIN5/style3.css [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+   position: relative;
+   border: 2px solid white;
+   background-color: white;
+   top: -147px;
+   left: -147px;
+   width: 636px;
+}
+.third {
+   position: relative;
+   border: 2px solid:black;
```
Comment (SnehaMakade) [inline vinod/html/ASSIN5/style3.css]: remove colon between solid and black

File: vinod/html/ASSIN6/assingment6.q2.html [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+            </div>
+            <fieldset>
+                <legend>Academic Details</legend>
+                <lable for="Skill set">Skill set:</lable>
+                <input type="checkbox" id="HTML" name="Skill Set" value="HTML" />
+                <label for="HTML">html</label>
+                <input type="checkbox" id="Java Script" name="Skill Set" value="Java Script" />
+                <label for="Java Script"> Java Script</label>
+                <input type="checkbox" id="Java" name="Skill Set" value="Java" />
+                <label for="Java"> Java</label>
```
Comment (SnehaMakade) [inline vinod/html/ASSIN6/assingment6.q2.html]: remove extra space

File: vinod/html/ASSIN6/assingment6.q2.html [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+                <input type="checkbox" id="Oracal" name="Skill Set" value="Oracal" />
+                <label for="Oracal"> Oracal</label><br /><br />
```
Comment (SnehaMakade) [inline vinod/html/ASSIN6/assingment6.q2.html]: remove all extra spaces

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do the cahnges as per comments given

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**